README.md

```diff
@@ -55,7 +55,7 @@ Input                     | Description
 `command`                 | <p>(Required) Script, function, or statement to execute. If the value of `command` is the name of a MATLAB script or function, do not specify the file extension. If you specify more than one script, function, or statement, use a comma or semicolon to separate them.<p/><p>MATLAB exits with exit code 0 if the specified script, function, or statement executes successfully without error. Otherwise, MATLAB terminates with a nonzero exit code, which causes the action to fail. To fail the action in certain conditions, use the [`assert`](https://www.mathworks.com/help/matlab/ref/assert.html) or [`error`](https://www.mathworks.com/help/matlab/ref/error.html) function.<p/><p>**Example:** `command: myscript`<br/>**Example:** `command: results = runtests, assertSuccess(results);`</p>
 `startup-options`         | <p>(Optional) MATLAB startup options, specified as a list of options separated by spaces. For more information about startup options, see [Commonly Used Startup Options](https://www.mathworks.com/help/matlab/matlab_env/commonly-used-startup-options.html).<p/><p>Using this input to specify the `-batch` or `-r` option is not supported.<p/><p>**Example:** `startup-options: -nojvm`<br/>**Example:** `startup-options: -nojvm -logfile output.log`</p>
 
-When you use this action, all of the required files must be on the MATLAB search path. If your script or function is not in the root of your repository, you can use the [`addpath`](https://www.mathworks.com/help/matlab/ref/addpath.html), [`cd`](https://www.mathworks.com/help/matlab/ref/cd.html), or [`run`](https://www.mathworks.com/help/matlab/ref/run.html) function to put it on the path. For example, to run `myscript.m` in a folder named `myfolder` located in the root of the repository, you can specify `command` like this:
+When you use this action, all of the required files must be on the MATLAB search path. If your script or function is not in the MATLAB current folder, you can use the [`addpath`](https://www.mathworks.com/help/matlab/ref/addpath.html), [`cd`](https://www.mathworks.com/help/matlab/ref/cd.html), or [`run`](https://www.mathworks.com/help/matlab/ref/run.html) function to put it on the path. For example, to run `myscript.m` in a folder named `myfolder` located in the MATLAB current folder, you can specify `command` like this:
```

Comment:
Hmm, I think I actually prefer keeping "root of your repository" and "root of the repository" here. It seems like it will be clearer for 99% of use cases (especially for less advanced users who are more likely to be reading the documentation), and if the user uses the `-sd` option they will instead get a "not found / not on path" error which hopefully cues them into understanding what the problem is.

Reply:
Thanks for your comment, @sameagen-MW. Do you feel the new link and note help clarify things?

Reply:
For run-command I'm a little bit less opposed because it's not as inconvenient to run `pwd`, but I still think there's probably a better alternative where we specify the default location in a different way.

I do quite like the addition on line 63 though!